docs/user-guide/aroma/sd-preparation.md

```diff
@@ -11,13 +11,13 @@ We will now place the required Aroma files on the SD Card.
 ### What You Need
 
 - The latest files from the [Aroma download page](https://aroma.foryour.cafe).
-    1. Scroll down to the **Download** section.
-    1. Read through the steps and click on the checkboxes.
-    1. Click on `Download Payloads` and `Download Base Aroma`.
 - The latest release of the [PayloadLoader Installer](https://github.com/wiiu-env/PayloadLoaderInstaller/releases).
 
 ### Instructions
 
+1. On the aroma site, scroll down to the **Download** section.
```

Comment:
```suggestion
1. On the Aroma site, scroll down to the **Download** section.
```
Also maybe put the link to the Aroma site here.

Reply:
So make it a hyperlink?

Reply:
yeah, at `Aroma` probably